src/pages/EventDetailsPage/index.tsx

```diff
@@ -41,11 +42,11 @@ function EventDetailsPage(): JSX.Element {
       </Grid>
 
       <Grid item xs={12}>
-        <CheckOffTable
-          getAttendances={() => getEventAttendances(eventId, true, false)}
-          checkOffAttendance={checkOffAttendance}
-          eventId={eventId}
-        />
+        {OfficerRenderPermission(CheckOffTable)({
+          getAttendances: () => getEventAttendances(eventId, true, false),
```

Comment:
Probably gonna be a merge conflict here, since u changed false to true in the other PR